object_store/src/client/get.rs

```diff
@@ -28,6 +28,12 @@ use reqwest::Response;
 pub trait GetClient: Send + Sync + 'static {
     const STORE: &'static str;
 
+    /// Configure the [`HeaderConfig`] for this client
+    const HEADER_CONFIG: HeaderConfig = HeaderConfig {
+        etag_required: true,
+        last_modified_required: true,
+    };
```

Comment:
I think using `Default` would make the intent clearer here

```suggestion
    /// Configure the [`HeaderConfig`] used for requests issued by this client
    const HEADER_CONFIG: HeaderConfig = HeaderConfig::default();
```

Reply:
Sadly this isn't possible as Default is not const